python/tests/test_genotypes.py

```diff
@@ -2215,3 +2215,8 @@ def test_variant_html_repr_no_site(self):
         html = v._repr_html_()
         ElementTree.fromstring(html)
         assert len(html) > 1600
+
+    def test_variant_repr(self, ts_fixture):
```

Comment:
Could we add some tests for the contents? For examples of how we done this elsewhere see `test_highlevel.py:3663` and `test_highlevel.py:2160`. Aim is to make a test that doesn't break if the data changes slightly, but that checks everything is there.

Reply:
What do you think of the tests I added?